### DISCUSSION TITLE
Weird behaviour when generating changelogs

### DISCUSSION THREAD

**Discussion by helpermethod** [jreleaser/jreleaser#230] [Q&A] [answered]
Hi!

I'm putting this here first because I don't know if it's a bug or intended behaviour:

When a commit title contains a tag, the tag doesn't show up in the generated changelog.

E.g.:

https://github.com/helpermethod/connect-offset-reset/releases/tag/v0.0.16

The second commit shows up as

    759c58e Fix bad character in

but when you click on the commit link you can see that the title is actually

    Fix bad character in <finalName>

Is the tag interpreted / substituted in some way?

Couldn't find anything in the docs.

**Comment by aalmiray** [comment, marked as answer]
I don't think JReleaser changes the title when a tag is present. You can check the locally generated changelog at `out/jreleaser/release/CHANGELOG.md`. If the tag is present then it may be that GitHub's rendering mechanism is performing an extra task, similarly as it is with the short commit hash: by default JReleaser does not create commit links based on hashes but you can tell the links are visible when looking at the rendered changelog.
